nilearn/input_data/nifti_masker.py

```diff
@@ -227,7 +231,7 @@ def __init__(self, mask_img=None, sessions=None, smoothing_fwhm=None,
         # Mask is provided or computed
         self.mask_img = mask_img
 
-        self.sessions = sessions
+        self.runs = runs
```

Comment:
I'm wondering whether we should deprecate this attribute, because this could break user code relying on the "sessions" attribute of the object. WDYT?
Maybe by doing something like this:

```python
@property
def sessions(self):
    warnings.warn(("'sessions' attribute is deprecated and will "
                   "be removed in 0.9.0, use 'runs' instead."))
    return self.runs
```

Reply:
Good point - I haven't test out the impact of my change yet. 
I will try your suggested solution.

Reply:
I think we need to do the same thing for the `sample_mask` attribute which will also disappear. 

Maybe setting:

```python
self._sample_mask = sample_mask
```

in the constructor. And add:

```python
@property
def sample_mask(self):
    warnings.warn('deprecated...will be removed...')
    return self._sample_mask
```

WDYT?

Reply:
Certainly!